Library/Formula/sqlite.rb

```diff
@@ -21,6 +21,7 @@ class Sqlite < Formula
   option "with-unlock-notify", "Enable the unlock notification feature"
   option "with-icu4c", "Enable the ICU module"
   option "with-functions", "Enable more math and string functions for SQL queries"
+  option "with-dbstat", "Enable the \"dbstat\" virtual table"
```

Comment:
No strong reason, I copied the sentence from the sqlite change listing and kept the double-quotes. I'll update to single ones.